src/deavinaSJun20/CheckArmstrong.java

```diff
@@ -0,0 +1,30 @@
+package deavinaSJun20;
+
+import java.util.Scanner;
+
+public class CheckArmstrong {
+
+	public static void main(String[] args) {
+		Scanner scanner = new Scanner(System.in);
+		System.out.println("Please Enter the number : ");
+		int number = scanner.nextInt();
+		System.out.println("Entered Number: " + number);
+		new CheckArmstrong().checkArmstrongNumber(number);
+		scanner.close();
+	}
+
+	void checkArmstrongNumber(int num) {
+		int temp = 0;
+		int num1 = num;
+		int sum = 0;
+		while (num != 0) {
+			temp = num % 10;
```

Comment:
Minimize variables. Here "temp" variable is not required. Logic is correct.

Reply:
updated file checked in as per review comments